internal/cluster/validations/validations.go

```diff
@@ -976,3 +978,30 @@ func ValidateArchitectureWithPlatform(architecture *string, platform *models.Pla
 
 	return nil
 }
+
+func ValidatePlatformCapability(platform *models.Platform, ctx context.Context, authzHandler auth.Authorizer, log logrus.FieldLogger) error {
+	if platform == nil {
+		return nil
+	}
+
+	var capabilityName *string
+	switch *platform.Type {
+	case models.PlatformTypeOci:
+		capabilityName = swag.String(ocm.PlatformOciCapabilityName)
+	}
+
+	if capabilityName == nil {
+		return nil
+	}
+
+	available, err := authzHandler.HasOrgBasedCapability(ctx, *capabilityName)
+	if err == nil && available {
+		return nil
+	}
+
+	if err != nil {
+		log.WithError(err).Errorf("error getting user %s capability", *capabilityName)
```

Comment:
Why is that log not inside `HasOrgBasedCapability`? I saw that it's duplicated 3 times now

Reply:
Done!